src/test/scala/cromwell/OptionalParamWorkflowSpec.scala

```diff
@@ -35,4 +37,35 @@ class OptionalParamWorkflowSpec extends CromwellTestkitSpec("OptionalParamWorkfl
       )
     }
   }
+
+  "A workflow with an optional parameter that has a prefix inside the tag" should {
+    "not include that prefix if no value is specified" in {
+      val wf = """
+         |task find {
+         |  command {
+         |    find ${File root} ${"-name " pattern?}
+         |  }
+         |}
+         |
+         |workflow wf {
+         |  call find
+         |}
+       """.stripMargin
+      val ns = WdlNamespace.load(wf, BackendType.LOCAL)
```

Comment:
For tests that are expected to pass, the loaded `wf: WdlSource` should be in `SampleWdl`, not inline above. I'm not sure about tests that are expected to fail, like this one. So, unless someone else can chime in with something better, I would leave it as is.

TL;DR: This is ok.